uniffi/Cargo.toml

```diff
@@ -5,20 +5,13 @@ authors = ["Firefox Sync Team <sync-team@mozilla.com>"]
 license = "MPL-2.0"
 edition = "2018"
 
-[[bin]]
-name = "uniffi-bindgen"
-path = "src/main.rs"
-
+# The dependency versions here have to agree with the corresponding ones in
+# uniffi_bindgen.
 [dependencies]
-weedle = "0.11"
+anyhow = "1.0"
 bytes = "0.5"
 ffi-support = "0.4"
-anyhow = "1.0"
-askama = "0.9"
-heck = "0.3"
-clap = "2.33"
-object = "0.20"
-serde = "1.0"
-bincode = "1.3"
-cargo_metadata = "0.10.0"
 lazy_static = "1.4"
+log = "0.4"
```

Comment:
These dependencies are re-exported for convenience.

Reply:
Yeah, +1 I like this a lot.